web-client/src/components/WebClientMap/CustomMapContol.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactNode, useEffect } from 'react';
+import { createPortal } from 'react-dom';
+
+const CustomMapControl = ({ map, controlPosition, children }: Props) => {
```

Comment:
I am not entirely sure what a "portal" is but I don't think that this should be called "CustomMapControl" and the other file should be called  "MyLocationControl"  These seem like very differnt files with very different functionalty and should have names that reflect their distinct functionality.

Reply:
I think this is actually a "MapControlContainer" and should be in the containers/ directory

Reply:
I think this is actually a CustomMapControlContainer because it acts containerish because it interacts with external state and dom

Reply:
or CustomMapControlPortal

Reply:
Controls are what the buttons on the maps are called.
This component allows you to add custom controls to the map as children to this component.
To me it makes sense to have the component named this because its purpose is to allow you to add custom map controls. 
But I would gladly change the name is someone can propose a better one.

Reply:
Ahh just got your other messages after I posted. I like your suggestions. Maybe container would be the best fit?

Reply:
Just figure out a way to distinguish "LocateMeControl" from "MapContainerControl".
By naming them with the same suffix, it looks like these have the same use case, but actualy one is a parent of the other.

Reply:
I see your point now. I will try to come up with better names later!